README.md

```diff
@@ -68,6 +68,22 @@ python3 MaxText/decode.py MaxText/configs/base.yml run_name=$YOUR_JOB_NAME
 ```
 Be aware, these decodings will be random. To get high quality decodings you need pass in a checkpoint, typically via the `load_parameters_path` argument.
 
+### Running on NVIDIA GPUs
+1. Clone MaxText.
+2. Pull a base container from NVIDIA's [JAX-Toolbox](https://github.com/NVIDIA/JAX-Toolbox/pkgs/container/jax-toolbox)
```

Comment:
If you strongly want to merge like this we can, but I'd prefer if we abstracted this into:
https://github.com/google/maxtext/blob/main/docker_build_dependency_image.sh

and eventually into:
https://github.com/google/maxtext/blob/main/maxtext_dependencies.Dockerfile#L2

Then your instructions could just be `bash docker_build_dependency_image.sh DEVICE=gpu`, that'd even run setup.sh and you'd be done!

Reply:
I'm actually working to do this in my PR #265. I'm trying to figure out a way 

https://github.com/google/maxtext/pull/265/files#diff-eb2b2f7e9b6a2cd1eebc3c05740e20d3006e9b966f3eefbf80d0bd7135f50c38

Reply:
This looks good to me @michelle-yooh. 
Ill remove the note from the README.